plugins/cisco_umbrella_destinations/plugin.spec.yaml

```diff
@@ -3,12 +3,13 @@ extension: plugin
 products: [insightconnect]
 name: cisco_umbrella_destinations
 title: Cisco Umbrella Destinations
-version: 4.0.0
+version: 5.0.0
 supported_versions: ["2023-03-29"]
```

Comment:
This can also be bumped if all the actions have ben tested to work with the current api